systemtest/src/test/java/io/strimzi/systemtest/metrics/MetricsST.java

```diff
@@ -112,6 +117,44 @@ void testKafkaConnectIoNetwork() {
         assertThat("Kafka Connect IO network count doesn't match expected value", values.stream().mapToDouble(i -> i).sum() > 0);
     }
 
+    @Test
+    void testKafkaExporterDataAfterExchange() throws Exception {
+        createTestMethodResources();
+        testMethodResources().deployKafkaClients(CLUSTER_NAME + "-" + Constants.KAFKA_CLIENTS).done();
+
+        availabilityTest(5000, CLUSTER_NAME, TEST_TOPIC_NAME);
+
+        kafkaExporterMetricsData = collectKafkaExporterPodsMetrics();
+        assertThat("Kafka Exporter metrics should be non-empty", kafkaExporterMetricsData.size() > 0);
```

Comment:
What do you think about this change? 
assertThat("Kafka Exporter metrics should be non-empty", kafkaExporterMetricsData.size() > 0);
```suggestion
        int actualValue =  kafkaExporterMetricsData.size();
        assertThat("Kafka Exporter metrics should be non-empty", actualValue, greaterThan(0);
```

Reply:
I am ok with `greaterThan()` but not sure if there is any value in putting metrics data size into variable. Or it's some java convention ?

Reply:
it would be nice`create` some `generic type of variables actualValue and exceptedValue` and use in the hamcrest asserts in the future. More here -> https://www.petrikainulainen.net/programming/testing/junit-5-tutorial-writing-assertions-with-hamcrest/